projects/element-ng/tabs-next/si-tab-next-link.component.ts

```diff
@@ -20,35 +21,31 @@ import { SiTabNextBaseDirective } from './si-tab-next-base.directive';
   styleUrl: './si-tab-next.component.scss',
   providers: [{ provide: SiTabNextBaseDirective, useExisting: SiTabNextLinkComponent }],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {
-    '[class.active]': 'routerLinkActive.isActive',
-    '[attr.aria-selected]': 'routerLinkActive.isActive'
-  },
   hostDirectives: [
     {
       directive: RouterLinkActive
     }
   ]
 })
 export class SiTabNextLinkComponent extends SiTabNextBaseDirective {
-  private router = inject(Router, { optional: true });
-  /** @defaultValue false */
-  override readonly active = signal(false);
+  private router = inject(Router);
   /** @internal */
-  routerLink = inject(RouterLink, { optional: true, self: true });
+  routerLink = inject(RouterLink, { self: true });
```

Comment:
Doesn't the removal of the `optional` pose a breaking change? The tabs used to be usable without hard dependency on the Angular router, e.g. for minimal use-cases where applications were not using a router at all.

Reply:
This is only the si-tab-next-link, which is designed to be used with the router. The "normal" `si-tab-next` still works without the router.

Reply:
@spike-rabbit yep, this sounds all logical, but technically seen, it's still a breaking change. But considering that the component wasn't released yet and indeed is called `link`, I'm perfectly fine to violate semantics in this case.